docs/kafka/nodejs/README.adoc

```diff
@@ -229,7 +229,7 @@ $ npm install
 $ node consumer.js
 ----
 +
-You should see the Node.js application start to run and connect to the Kafka instance. However, because you haven't yet run the producer component, the consumer has no country names to display.
+You should see see the Node.js run and connect to the Kafka instance. However, because you haven't yet run the producer component, the consumer has no country names to display.
```

Comment:
@fmcdonal Repeat of the word 'see' here.

Also, it should read: "You should see the Node.js _application_ run...". This one is my fault :) I wrote it incorrectly in the JIRA, which I just updated.